### PR TITLE
DDP-4798: Setup new pediatric BrainCancerProject Angular app

### PR DESCRIPTION
**Pull request by fat23cat**
Erin presented new BrainCancerProject design - https://projects.invisionapp.com/d/main/#/projects/prototypes/20209715
The new design is radically different from the old one, so, IMO, it would be easier to create new Angular app and new study, because the study also contains styles and new design assumes new count-me-in page. This will make for easier development, and keep open the possibility for concurrent changes to what we currently have for the brain.
And when redesign done, we can just remove the old app and the study, and then rename and use the new app and the study.

I added many reviewers, I'm not expecting real code review, I just want to discuss the idea, maybe someone can suggest something else.

### CODE REVIEW

File: config/pepperConfig.js.ctmpl [hunk 1]
```diff
@@ -95,6 +95,22 @@ var DDP_ENV = {
     {{end}}
 {{end}}
 
+{{if eq $study_guid "cmi-brain-v2"}}
```
Comment (fat23cat) [inline config/pepperConfig.js.ctmpl]: New study guid.

File: ddp-workspace/angular.json [hunk 2]
```diff
@@ -1155,6 +1155,148 @@
           }
         }
       }
+    },
+    "ddp-brain-v2": {
```
Comment (fat23cat) [inline ddp-workspace/angular.json:1159]: New Angular app. `ddp-brain` is still alive.

File: config/pepperConfig.js.ctmpl (hunk 1 above)
Comment (yufengwng) [reply config/pepperConfig.js.ctmpl]: Creating new app on frontend is a good idea. Creating new study on backend will be bad... A lot of backend configuration, users, and data is tied to the `cmi-brain` study. Moving it to a new study `cmi-brain-v2` is like doing a whole study migration...

Comment (fat23cat) [reply config/pepperConfig.js.ctmpl]: Hmm, can we create custom activities in the current study scope and return them only for specific test users?

Comment (yufengwng) [reply config/pepperConfig.js.ctmpl]: Might be a bit tricky but certainly do-able.

**Review by MocanaAtBroad — APPROVED** [review]
Let's get it into develop!

**Pull request merged**